### ISSUE TITLE
A report not yet executed is marked like a success

### ISSUE THREAD

**Issue by xavier-GitHub76** [apache/superset#29622]
Hello

A report not yet executed is marked like a success

## Screenshot

![image](https://github.com/user-attachments/assets/14dfcbac-ae74-4829-9634-3c98788fb363)

Another symbol could be used for this state : 
![not_yet_executed](https://github.com/user-attachments/assets/7324fa5e-fb71-40c9-8fb8-5a6274ad49e2)

Best regards

**Comment by rusackas** [comment]
Agreed, that is confusing. Tagging @kasiazjc for design/icon input, and @yousoph to see if someone can take it under their wing.

**Comment by rusackas** [comment]
This has gone silent for 250+ days. Is this still an issue in current (5.0.0/master) Superset?

**Comment by xavier-GitHub76** [comment]
The issue is still in 5.0.0-RC2

**Comment by rusackas** [comment]
This seems like a very easy change (you can just search the codebase for "nothing triggered" and you'll find it... but the state seems a bit indeterminate. Anyone have a favorite icon [from the set](https://ant.design/components/icon/)?

**Comment by yousoph** [comment]
Maybe CalendarOutlined ?

**Comment by rusackas** [comment]
Still very much a thing — confirmed on current master. The root cause is that a report/alert that's never run defaults to `last_state = "Not triggered"`, and the status icon rendered that with a green checkmark + success color, so it reads as "succeeded."

Put up #41121 to give the not-yet-run state a neutral calendar icon (thanks for the `CalendarOutlined` nudge @yousoph) instead of the green check, with a "Report not yet run" tooltip for reports. Added a test so it doesn't regress.
